### PR TITLE
Nt 10

### PR DESCRIPTION
**Pull request by javadrip**
Hotspot issue is resolved by adding the `.crop("focalpoint")` transformer to the URL builder.

### CODE REVIEW

**Comment by javadrip** [comment]
Add alt to mainImage, resolving an error where "Type '{}' is not assignable to type 'string'."

File: types/Post.ts [hunk 1]
```diff
@@ -16,12 +16,8 @@ export type Post = {
   categorySlug: string;
   excerpt: string;
   author: Author;
-  mainImage: {
-    alt?: string;
-    asset: {
-      _ref: string;
-      url: string;
-    };
+  mainImage: Image & {
```
Comment (sashabaranova) [inline types/Post.ts:19]: change to using `extends` as discussed on Discord

**Comment by javadrip** [comment]
@sashabaranova okay if I go ahead and merge this PR?

**Comment by sashabaranova** [comment]
@javadrip yes sure

**Pull request merged**